src/config/AppPermissions.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+/*
+ * Defines the mapping between:
+ * - roles application defined in core Enterprise Application
+ * - WebApplication persona
+ */
+const APP_ROLES = {
+  organizationAdmin: 'Organization.Admin',
+  organizationCollaborator: 'Organization.Collaborator',
+  organizationModeler: 'Organization.Modeler',
+  // powerUser
+  organizationUser: 'Organization.User',
+  // viewer
+  organizationViewer: 'Organization.Viewer',
+  platformAdmin: 'Platform.Admin',
+};
```

Comment:
Do we need to put this mapping in the config for integrators ?
If these constants are defined by the back-end / cloud configuration, we should probably move it to `src/services/config/ApiConstants.js` because integrators are not supposed to modify these values

Reply:
Changes have been done regarding our discussion ;)